jest.config.js

```diff
@@ -0,0 +1,7 @@
+const { jestConfig } = require('@salesforce/lwc-jest/config');
+module.exports = {
+    ...jestConfig,
+    moduleNameMapper: {
+        '^lightning/navigation$': '<rootDir>/force-app/test/jest-mocks/lightning/navigation',
```

Comment:
Doesn't seem like there's really a precedent here for where to put custom mocks. `force-app/test/jest-mocks` made sense to me but I'm open to suggestions.

Reply:
I think that's a good name. Customers can still change (i. e. I used `jest-apex-mocks`). Minimum should be that we separate it, like @trevor-bliss did, from the metadata folders.